configure

```diff
@@ -2214,10 +2216,10 @@ check_higgstools_libs() {
 	         fi
             if [ -z "$higgstools_lib_dir" ]; then
                 # assume that the linker will look in the default paths
-                HIGGSTOOLSLIBS="-Wl,-Bdynamic -$HIGGSTOOLSLIBS_"
+                HIGGSTOOLSLIBS="model_specific/SM/libmodel_specific_SM.a -Wl,-Bdynamic -$HIGGSTOOLSLIBS_"
             else
                 # assume that the linker will look in the default paths
-                HIGGSTOOLSLIBS="-L$higgstools_lib_dir -Wl,-rpath,$higgstools_lib_dir,-Bdynamic $HIGGSTOOLSLIBS_"
+                HIGGSTOOLSLIBS="model_specific/SM/libmodel_specific_SM.a -L$higgstools_lib_dir -Wl,-rpath,$higgstools_lib_dir,-Bdynamic $HIGGSTOOLSLIBS_"
```

Comment:
Same here: We should have 
~~~
HIGGSTOOLSLIBS="model_specific/SM/libmodel_specific_SM.${extension} -L$higgstools_lib_dir -Wl,-rpath,$higgstools_lib_dir,-Bdynamic $HIGGSTOOLSLIBS_"
~~~
if I understand correctly.